.github/workflows/publish_pypi.yml

```
@@ -2,7 +2,7 @@ name: Publish Icesat2 to PyPI
 
 on:    
   release:
-    types: [published]
+    types: [prereleased]
```

I think this should stay as "published" 
```suggestion
    types: [published]
```
... which catches both pre-releases and releases.

Whether it's marked as pre-release or not on pypi depends on the version number – whether it has an alpha, beta or (release) candidate version number (I think).

---

Done